src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+import duke.task.ToDo;
+import org.junit.jupiter.api.Test;
+
```

Comment:
I believe the static imports should be at the top based on coding standard (although not 100% sure)